cdk/dcm_floodlight/transform.js

```diff
@@ -1,10 +1,10 @@
 /* eslint-disable no-param-reassign */
 const get = require("get-value");
 const {
-  getValueFromMessage,
   removeUndefinedAndNullValues,
   isDefinedAndNotNull
 } = require("rudder-transformer-cdk/build/utils");
+const logger = require("../../logger");
 const { CustomError, getIntegrationsObj, isEmpty } = require("../../v0/util");
 
 // append properties to endpoint
```

Comment:
You can use `join` here example
```
map().join('operator')
```

Reply:
when you do `return ${key}=${payload[key]}` from map after that you append those in string function

Reply:
![Screenshot 2022-06-07 at 8 19 51 PM](https://user-images.githubusercontent.com/25389167/172411024-726e4bca-4b6a-4151-9b48-9643e5e0c388.png)